pavelib/prereqs.py

```diff
@@ -159,6 +159,7 @@ def install_node_prereqs():
     "edxval",                       # Because it was bork-installed somehow.
     "django-storages",
     "django-oauth2-provider",       # Because now it's called edx-django-oauth2-provider.
+    "django-pipeline",              # This does not need to be editable.
```

Comment:
There's no need to uninstall the package just to remove the -e.  If you are seeing problems without the uninstall, we should debug it.